.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
Is this part of your change? I don't think so. Probably need to rebase.

Reply:
I'll try that Sunday:)

Reply:
worked :)